src/components/SearchPhotos/SearchPhotos.js

```diff
@@ -49,15 +54,20 @@ class SearchPhotos extends React.Component {
           <div>There are no results for {searchTerm}.</div>
         )}
         {readyWithPhotos && (
-          <Container>
-            {photoData.results.map((photo) => {
-              return (
-                <StyledLink key={photo.id} to={`/photo/${photo.id}`}>
-                  <StyledImage src={photo.urls.small} alt={photo.id} />
-                </StyledLink>
-              );
-            })}
-          </Container>
+          <StyledResponsiveMasonry
+            columnsCountBreakPoints={{ 350: 1, 900: 2, 1285: 3 }}
```

Comment:
you used this in a previous component, right? you can make a file that export this object and you can import it where you use it

Reply:
Like this?
`import {columnBreaks} from "../../utils/variables"`

`<StyledResponsiveMasonry`
 ` columnsCountBreakPoints={columnBreaks}`
  `gutter="0"`
`>`